Gemfile

```diff
@@ -9,4 +9,5 @@ group :development, :test do
   gem "webmock", "~> 3.14"
   gem "factory_bot", "~> 6.2"
   gem "facets", "~> 3.1"
+  gem "typhoeus", "~> 1.0"
```

Comment:
What is this dependency for? Did this get changed in the upstream generator?

Reply:
I believe it is not needed because typhoeus is a part of the freeclimb gem package, this is removed in latest commit.